<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Export_Window</class>
 <widget class="QWidget" name="Export_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>83</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export_Window</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>324</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="export_image_button">
      <property name="text">
       <string>Export as image</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="export_mat_button">
      <property name="text">
       <string>Export as .mat</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="export_txt_button">
      <property name="text">
       <string>Export as .txt</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
